cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import requests
+
+from cg.clients.freshdesk.constants import EndPoints
+from cg.clients.freshdesk.models import Ticket
+
+LOG = logging.getLogger(__name__)
+
+
+class FreshdeskClient:
+    """Client for communicating with the freshdesk REST API."""
+
+    def __init__(self):
+        self.api_key = None
+        self.url = None
+
+    def init_app(self, url, api_key):
+        """Set up the client."""
+        self.url = url
+        self.api_key = api_key
```

Comment:
why do you need this here? isn't it enough to assign it in the __init__()?

Reply:
Yes, this is not very logical... See this comment: https://github.com/Clinical-Genomics/cg/pull/3317#discussion_r1627195681